apps/auth0/scripts/dependencies-sync.ts

```diff
@@ -0,0 +1,40 @@
+import tools from 'auth0-deploy-cli';
+import { readFileSync, writeFileSync } from 'fs';
+import yaml from 'js-yaml';
+import packageJson from '../package.json';
+
+const main = async () => {
+  const fileName = 'tenant.yaml';
+  try {
+    const yamlFile = readFileSync(`./${fileName}`, 'utf8');
+    const data = yaml.load(yamlFile) as Parameters<
+      typeof tools['tools']['deploy']
+    >[0];
+
+    const actions = data.actions?.map((action) => {
+      const dependencies = action.dependencies?.map(
+        (dependency: { name: keyof typeof packageJson.dependencies }) => {
+          const version = packageJson.dependencies[dependency.name];
+          return version
+            ? {
+                ...dependency,
+                version,
+              }
+            : dependency;
```

Comment:
I believe this means that if the dependency isn't listed in package json we just leave the dependency without the version - does it work that way? Maybe we should throw if the dependency isn't listed?

Reply:
Yeah, it gets latest